envs/conda-forge.yml

```diff
@@ -0,0 +1,27 @@
+# This environment is intended to match the conda-forge improver-feedstock at
+# https://github.com/conda-forge/improver-feedstock
```

Comment:
After re-reading the YAML above and your comment on it, this is not well written and I should reword it.
My intention is that that the conda-forge feedstock should get updated to match any changes here.

I think pulling in the dependencies from the current conda-forge package would be difficult.
For example, if we add a new dependency in the future, a CI job that used `improver --only-deps` to get the dependencies from the current conda-forge package would fail due to that new dependency being missing, until we make a new tag in this repository and a new release on conda-forge that does include the dependency.

From reading the conda-forge docs, I can't see any neat way of the feedstock `meta.yaml` picking up the dependency list from this repository, so would be happy to see examples or suggestions if a good way of avoiding the duplication does exist.

Reply:
Addressed in 1774aea